### DISCUSSION TITLE
Allow Gemini access via Google AI (not Vertex AI)

### DISCUSSION THREAD

**Discussion by hassiebp** [langfuse/langfuse#4669] [Ideas]
### Describe the feature or potential improvement

Gemini models can be accessed via Vertex AI (already possible in Langfuse) and Google AI (not yet possible). Add support for Google AI access via a single API key only. 

### Additional information

https://cloud.google.com/vertex-ai/generative-ai/docs/migrate/migrate-google-ai#google-ai

**Comment by marcklingen** [comment]
Completed, https://langfuse.com/changelog/2025-02-25-google-ai-studio-support
